Resources/Prototypes/Entities/Objects/Misc/paper.yml

```diff
@@ -103,10 +103,10 @@
     maxWritableArea: 400.0, 0.0
 
 - type: entity
-  name: captain's thoughts
+  name: station representative thoughts
   parent: Paper
   id: PaperCaptainsThoughts
-  description: "A page of the captain's journal. In luxurious lavender."
+  description: "A page of the station representative journal. In luxurious lavender."
```

Comment:
this isnt great either because the paper itself is I believe a texture that has 'captains thoughts' or similar on it as well in the background, so it will still mismatch

Reply:
Planed to update it later, ill un-comment it for now